bootstrap-orbit.sh

```diff
@@ -95,18 +97,13 @@ if [ $? -ne 0 ]; then
 else
   echo "Found conan. Checking version..."
   CONAN_VERSION="$(conan --version | cut -d' ' -f3)"
-  CONAN_VERSION_MAJOR="$(echo "$CONAN_VERSION" | cut -d'.' -f1)"
-  CONAN_VERSION_MINOR="$(echo "$CONAN_VERSION" | cut -d'.' -f2)"
-
-  CONAN_VERSION_MAJOR_REQUIRED=1
-  CONAN_VERSION_MINOR_MIN=36
-
-  if [ "$CONAN_VERSION_MAJOR" -eq $CONAN_VERSION_MAJOR_REQUIRED -a "$CONAN_VERSION_MINOR" -lt $CONAN_VERSION_MINOR_MIN ]; then
+  
+  if [[ "$CONAN_VERSION" < "$CONAN_VERSION_REQUIRED" ]]; then
```

Comment:
Hmmm, I believe this is only doing lexicographical comparison, meaning `"1.5" < "1.42"` is false.

I guess it's the same for powershell. I'd love to have a solution here. The only hack I could quickly come up with was the major/minor split thing.

Reply:
You are right. I did not think about this case. Will fix it